### PR TITLE
Add search by name

### LINKED ISSUE #1220: Search in Profiles management

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #1220

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
https://github.com/user-attachments/assets/4ebebc90-c638-4dc6-a160-f96e5233c7f9

File: src/components/ProfilesManagementLayout/ProfilesManagementLayout.i18n/ru.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
     "Scheduled deactivations": "Запланированные деактивации",
     "Access coordination": "Согласование доступов",
     "Requests": "Заявки",
-    "Decrees": "Декреты"
+    "Decrees": "Декреты",
+    "Search in the table": ""
```
Comment (tsumo) [inline src/components/ProfilesManagementLayout/ProfilesManagementLayout.i18n/ru.json:7]: translation

File: src/components/ProfilesManagementSearch/ProfilesManagementSearch.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { FiltersSearchContainer } from '@taskany/bricks';
+
+import { SearchFilter } from '../SearchFilter';
+import { useUserListFilter } from '../../hooks/useUserListFilter';
+
+import { tr } from './ProfilesManagementSearch.i18n';
+
+export const ProfilesManagementSearch = () => {
```
Comment (tsumo) [inline src/components/ProfilesManagementSearch/ProfilesManagementSearch.tsx:8]: Unused component

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/modules/userTypes.ts [hunk 3]
```diff
@@ -76,7 +76,9 @@ export interface UserFilterQuery {
     supervisors?: string[];
     activity?: 'active' | 'inactive';
 }
-
+export interface UserFilterQueryd {
```
Comment (asabotovich) [inline src/modules/userTypes.ts:79]: typo

**Review by tsumo — APPROVED** [review]

**Review by nyet-ty — APPROVED** [review]

**Review by asabotovich — APPROVED** [review]

**Pull request merged**